gratipay/utils/emails/queue_branch_email.py

```diff
@@ -7,6 +7,23 @@
 from gratipay import wireup
 
 
+def get_recently_active_participants(db):
+    return db.all("""
+        SELECT p.*::participants
```

Comment:
not familiar with this :: syntax. could you please share a reference about its usage?